lib/enhancements/edition.rb

```diff
@@ -87,4 +87,8 @@ def register_with_panopticon
   def notify_publishing_api
     PublishingAPINotifier.perform_async(self.id.to_s)
   end
+
+  def self.edition_types
+    [AnswerEdition, GuideEdition, TransactionEdition, PlaceEdition, ProgrammeEdition, LicenceEdition, VideoEdition, BusinessSupportEdition, SimpleSmartAnswerEdition, CompletedTransactionEdition, HelpPageEdition, CampaignEdition]
```

Comment:
is this a manually curated list or is this the list of all edition subtypes? Is `LocalTransactionEdition` deliberately omitted?

It'd be good to not have to change this line if a new format was added or an existing one removed; Rails has [a method](http://apidock.com/rails/v4.2.1/Class/subclasses) for getting subclasses easily, so we could use `Edition.subclasses` in place of `Edition.edition_types`.